src/configuration/parser.rs

```diff
@@ -1,75 +1,9 @@
 //! This module is responsible for creating a default configuration file as well as validating and reading the config
 //! file.
 //!
-//! A default configuration file looks like this:
+//! You can find the default config in `config_template.toml`.
 //!
-//! ```toml
-//! [netbox]
-//! netbox_uri = ""
-//! netbox_api_token = ""
-//!
-//! [system]
-//! name = "some_name" # Required for virtual machines!
-//! site_id = 0 # The ID of the site this device is located at.
-//! description = ""
-//! comments = "Automatically registered using Nazara."
-//! device_type = 0
-//! role = 0
-//! # Name of the network interface to set. (e.g eth0, etc)
-//! # If not set, the first active interface will be selected.
-//! primary_network_interface = ""
-//! face = "" # Direction this device may face (e.g front or rear)
-//! status = "active" # Status of the device. 'active' by default.
-//! airflow = "front-to-rear" # Direction of airflow.
-//!
-//! # Optional data of your device
-//! # This section may be empty
-//! [[system.optional]]
-//! # tenant_group = 0 # The ID of the department this device belongs to.
-//! # tenant = 0 # ID of the team or individual this device blongs to.
-//! # location = 0 # ID of the location of the device.
-//! # rack = 0 # ID of the Rack this device sits in.
-//! # position = 0 # Position of the device within the Rack.
-//! platform = "x86_64" # Name of the paltform of this device.
-//!
-//! # These will be parsed into a single HashMap. You must provide
-//! # the correct field labels as there is no way for Nazara to know.
-//!
-//! # These values are purely exemplary.
-//! [system.custom_fields]
-//!
-//! # Network Interfaces Configuration (optional)
-//! #[[nwi]]
-//! #name = "" # Required. Must match interface that exists on the machine.
-//! #enabled = true
-//! #rtype = "type1"
-//! #parent = 1
-//! #bridge = 1
-//! #lag = 1
-//! #mtu = 1500
-//! #duplex = "full"
-//! #wwn = "wwn12345"
-//! #mgmt_only = false
-//! #description = "Automatically created by Nazara."
-//! #mode = ""
-//! #rf_role = ""
-//! #rf_channel = ""
-//! #poe_role = ""
-//! #poe_channel = ""
-//! #rf_channel_frequency = 2400.0
-//! #rf_channel_width = 20.0
-//! #tx_power = 20
-//! #untagged_vlans = [10, 20]
-//! #tagged_vlans = [30, 40]
-//! #mark_connected = true
-//! #wireless_lans = [50, 60]
-//! #vrf = 1
-//! # Custom fields specific for this interface
-//! #[nwi.custom_fields]
-//! # ...
-//! ```
-//!
-//! It will be created at ` $HOME/.nazara/.config.toml`.
```

Comment:
I get you hate long docstrings, I really do. But I showed the config template here so it is fully displayed on the web docs. If you can embed it thats fine otherwise I would like to keep it here.